index.html

```diff
@@ -9,7 +9,6 @@
   <body>
     <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
 
-   
     <script  src="js/index.js"></script>
```

Comment:
I suggest you leave your script tag at the bottom of your code as per proper practice